src/Grid/Items/DataGridItem.php

```diff
@@ -49,7 +49,7 @@ public final function setIdentifier($identifier)
      * @return boolean true on success or false on failure.
      * The return value will be casted to boolean if non-boolean was returned.
      */
-    public final function offsetExists($offset)
+    public final function offsetExists(mixed $offset): bool
```

Comment:
mixed type-hint support added in PHP8, but in composer.json still PHP7.1 support